src/markers/MarkerManager.js

```diff
@@ -0,0 +1,427 @@
+import DragPan from 'ol/interaction/DragPan';
+import Overlay from 'ol/Overlay';
+import VectorLayer from 'ol/layer/Vector';
+import 'ol/ol.css';
+import VectorSource from 'ol/source/Vector';
+import Style from 'ol/style/Style';
+import Stroke from 'ol/style/Stroke';
+import Collection from 'ol/Collection';
+import Feature from 'ol/Feature';
+
+import { getUnitsSuffix } from './utils';
+import { getShortestLineBetweenOverlayAndFeature } from './utils';
+
+const MapEvents = {
+  POINTER_MOVE: 'pointermove',
+  POINTER_UP: 'pointerup'
+};
+
+class MarkerManager {
+  constructor({
+    map,
+    source,
+    geometries,
+    unlinkGeometries = [],
+    undraggableGeometries = [],
+    formatters,
+  } = {}) {
+    this._markers = new Map();
+    this._listeners = new Map();
+    this._links = new Collection([], { unique: true });
+
+    this._unlinkGeometries = unlinkGeometries;
+    this._undraggableGeometries = undraggableGeometries;
+    this._geometries = geometries;
+    this._formatters = formatters;
+    this._map = map;
+    this._source = source;
+
+    const styleTag = document.createElement('style');
+    styleTag.innerHTML = `
+      .ol-tooltip {
+        color: #9ccef9;
```

Comment:
This was updated in recent changes.

Reply:
Are you sure? It looks like the default color is used. When the style of a roi gets set, it will not be applied to the text of measurement markups. It's a minor thing, we can also improve in a future version.

Reply:
Maybe it's something on my side then. I'm always changing and testing with my OHIF fork which sets highlighting colors when hovering the annotations. I'll take a look.

Reply:
Thanks. This is non-blocking. In fact, it may be better to not color the markup text along with the marker. Instead, we could put a background box behind the text to ensure the text can be read independent of image content (staining, etc.). What do you think?